src/components/Layout/Navbar/Account/index.tsx

```diff
@@ -31,7 +31,7 @@ const Account = () => {
     accessTokenServices?.services?.maiarId?.removeToken?.();
     localStorage.clear();
     sessionStorage.clear();
-    logout(routeNames.home, (route) => navigate(route!));
+    logout(location.origin + routeNames.home, (route) => navigate(route!));
```

Comment:
skip the first argument from redirect function and just navigate(routeNames.home), this won't be called on wallet